npm install failing with Invalid tag name for fundamental-ngx > 0.33.0

#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
When running npm install the command fails due to an invalid tag.
```
npm ERR! code EINVALIDTAGNAME
npm ERR! Invalid tag name "^13,.0.0": Tags may not have any characters that encodeURIComponent encodes.
```

After investigating the logs it seems like that this URL https://registry.npmjs.org/@fundamental-ngx%2fcore is the culprit. After opening the link and searching for `^13,.0.0` it seems like that string somehow got included in the peer dependencies:
![Screenshot 2022-02-15 at 17 13 53](https://user-images.githubusercontent.com/19818061/154102604-6fbe961d-cf04-4e78-afca-4f3d55d9a6b7.png).

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 13
fundamental-ngx 0.33.0 and later

#### If this is a bug, please provide steps for reproducing it.
- add fundamental-ngx > 0.33.0 as a dependency
- delete `package.json` and `node_modules`
- run `npm install`

#### Please provide relevant source code if applicable.
Had a quick browse through the code, but wasn't able to find the wrong version (probably set via `ANGULAR_VER_PLACEHOLDER` or something).

#### Is there anything else we should know?
n.a.

--- a/ci-scripts/sync-version.sh
+++ b/ci-scripts/sync-version.sh
@@ -10,7 +10,7 @@ NEW_VERSION=$(node -p "require('./package.json').version")
 echo "Updating packages.json under dist/libs with version ${NEW_VERSION}"
 
 # As Angular version listed as peerDependency it should be ^X.0.0 to support any minor version
-ANGULAR_VERSION=$(node -p "'^' + require('./package.json').dependencies['@angular/core'].match(/\d+/).concat('.0.0')" )
+ANGULAR_VERSION=$(node -p "'^' + require('./package.json').dependencies['@angular/core'].match(/\d+/)[0].concat('.0.0')" )
 RXJS_VERSION=$(node -p "require('./package.json').dependencies['rxjs']")
 FAST_DEEP_EQUAL_VERSION=$(node -p "require('./package.json').dependencies['fast-deep-equal']")
 FDSTYLES_VERSION=$(node -p "require('./package.json').dependencies['fundamental-styles']")

